PWGJE/EMCALJetTasks/macros/AddTaskRecoilJetYield.C

```diff
@@ -21,10 +21,10 @@ AliAnalysisTaskRecoilJetYield* AddTaskRecoilJetYield(const char * njetsData, //d
 								    TString     trigClass      = "",
 								    TString     kEmcalTriggers = "",
 								    TString     tag            = "",
-								    AliAnalysisTaskRecoilJetYield::JetShapeType jetShapeType,
-								    AliAnalysisTaskRecoilJetYield::JetShapeSub jetShapeSub,
-								    AliAnalysisTaskRecoilJetYield::JetSelectionType jetSelection,
-								    Float_t minpTHTrigger =0.,  Float_t maxpTHTrigger =0., AliAnalysisTaskRecoilJetYield::DerivSubtrOrder derivSubtrOrder = 0  ) {
+								    AliAnalysisTaskRecoilJetYield::JetShapeType jetShapeType = kTrue,
+								    AliAnalysisTaskRecoilJetYield::JetShapeSub jetShapeSub = kNoSub,
+								    AliAnalysisTaskRecoilJetYield::JetSelectionType jetSelection = kInclusive,
```

Comment:
Still not OK: It needs to be AliAnalysisTaskRecoilJetYield::kTrue, AliAnalysisTaskRecoilJetYield::kNoSub and AliAnalysisTaskRecoilJetYield::kInclusive, otherwise they are interpreted as coming from the global namespace which they aren't. In addition please be careful with too general names - kTrue was a risky choice for that enum value.

Reply:
Hi @haandrew,

Thanks for fixing! I expect now your wagons should work under ROOT6 - did you try locally?  Just approved the PR.

Cheers

Markus